tools/build.js

```diff
@@ -8,6 +8,7 @@ import docs from '../docs/build';
 import dist from './dist/build';
 import { copy } from './fs-utils';
 import { setExecOptions } from './exec';
+import { exec } from '../tools/exec';
```

Comment:
This is in the same directory so this could be cleaned up with `import { exec } from './exec';`